.circleci/config.yml

```diff
@@ -143,6 +143,7 @@ commands:
             # that will also block dependencies from being installed.
             # as a result we first install dependencies by installing the
             # package and then immediately remove it again
+            pip install -r requirements-bundle.txt
```

Comment:
perhaps `requirements-test.txt` instead of `requirements-bundle.txt`?

Reply:
Solved